src/internal/deploy/cmds/cmds.go

```diff
@@ -537,7 +546,19 @@ func standardDeployCmds() []*cobra.Command {
 				// Serve the Pachyderm object/block API locally, as this is needed by
 				// our tests (and authentication is disabled anyway)
 				opts.ExposeObjectAPI = true
+
+				// Set the postgres and etcd nodeports explicitly for developers
+				opts.PostgresOpts.Port = 32228
+				opts.EtcdOpts.Port = 32379
```

Comment:
We were setting these nodeports in a kind of obscure way before, but I assumed the goal was to have fixed nodeports for developers so we can easily connect to postgres and etcd?